src/content/app/entity-viewer/variant-view/transcript-consequences/TranscriptConsequences.tsx

```diff
@@ -145,6 +128,88 @@ const PanelHeader = (props: {
   );
 };
 
+const TranscriptConsequencesPerGene = (props: {
+  genomeId: string;
+  variantId: string;
+  gene: TranscriptConsequencesData['geneData'][number];
+  variant: TranscriptConsequencesData['variant'];
+  allele: NonNullable<TranscriptConsequencesData['allele']>;
+  transcriptConsequences: NonNullable<
+    TranscriptConsequencesData['transcriptConsequences']
+  >;
+}) => {
+  const { genomeId, gene, variantId, variant, allele, transcriptConsequences } =
+    props;
+  const strand = gene.slice.strand.code;
+  const alleleSequence = allele.allele_sequence;
+
+  const transcriptConsequencesWithTranscript: Array<
+    (typeof transcriptConsequences)[number] & {
+      transcript: (typeof gene)['transcripts'][number];
+    }
+  > = [];
+
+  const sortedTranscripts = defaultSortForTranscripts(gene.transcripts);
+
+  for (const transcript of sortedTranscripts) {
+    // Currently, stable ids in transcript consequences returned by the api
+    // are unversioned; later on, they are expected to be changed to versioned ids.
+    const consequence = transcriptConsequences.find(
+      (cons) =>
+        cons.stable_id === transcript.stable_id ||
+        cons.stable_id === transcript.unversioned_stable_id
+    );
+
+    if (consequence) {
+      transcriptConsequencesWithTranscript.push({
+        ...consequence,
+        transcript
+      });
+    }
+  }
+
+  const transcriptConsCount = transcriptConsequencesWithTranscript.length;
+
+  return (
+    <div className={styles.geneSection}>
+      <div className={classnames(styles.row, styles.header)}>
+        <div className={styles.left}></div>
+        <div className={styles.middle}>
+          <div className={styles.headerMiddleColumn}>
+            <TranscriptAllele
+              referenceSequence={allele.reference_sequence}
+              alleleSequence={alleleSequence}
+              alleleType={allele.allele_type.value}
+              strand={strand}
+            />
+            <div className={styles.geneDetails}>
+              <span className={styles.label}>Gene</span>
+              {gene.symbol && (
+                <span className={styles.geneSymbol}>{gene.symbol}</span>
+              )}
+              <span className={styles.geneStableId}>{gene.stable_id}</span>
+            </div>
+          </div>
+        </div>
+        <div className={classnames(styles.right, styles.transcriptsCount)}>
+          <span>{`${transcriptConsCount} `}</span>
```

Comment:
Can't this be just `{transcriptConsCount}`?

Reply:
Yes; I initially had the content of this and the next span as a single string, until it turned out different parts of it needed different styling...

Removed the string interpolation.